content/shmem_collective_test.tex

```diff
@@ -0,0 +1,34 @@
+\apisummary{
+    The routine outputs the status of the operation identified by the request.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+int @\FuncDecl{shmem\_req\_test}@(shmem_req_h request);
+\end{Csynopsis}
+
+\begin{apiarguments}
+
+  \apiargument{IN}{request}{Request handle}
+
+\end{apiarguments}
+
+\apidescription{
+    A call to \FUNC{shmem\_req\_test} returns immediately. If the
+    operation identified by the request is completed, it returns
+    zero, and the request object is deallocated and cannot be reused.
```

Comment:
Could request objects have value `SHMEM_REQUEST_INVALID` after they are deallocated?  In which case this routine and `shmem_req_wait` would take an argument of type `shmem_req_h *`.

Reply:
@davidozog Just updated. Please let me know if it's missing something.